libticables/trunk/configure.ac

```diff
@@ -161,7 +161,7 @@ case "$host" in
   m68k-*-linux-*)        ARCH="-D__M68K__ -D__LINUX__" ;;
   mips*-*-linux-*)       ARCH="-D__MIPS__ -D__LINUX__" ;;
   powerpc-*-linux-*)     ARCH="-D__PPC__  -D__LINUX__" ;;
-  powerpc64-*-linux-*)   ARCH="-D__PPC__  -D__LINUX__" ;;
+  powerpc64*-*-linux-*)   ARCH="-D__PPC__  -D__LINUX__" ;;
   powerpc64le-*-linux-*) ARCH="-D__PPC__  -D__LINUX__" ;;
```

Comment:
This line should be matched by the previous one, so you should be able to remove it.

Reply:
Not sure, as it seems to be needed for 'powerpc64el'. (But line 165 isn't needed anymore with 'powerpc64*-*-linux-*)' in place, I guess).

Reply:
oh, I have to learn this githup workflow ;-/